coordinator/src/block_streams_handler.rs

```
@@ -1,3 +1,5 @@
+#![cfg_attr(test, allow(dead_code))]
```

Since the `impl` is mocked in `test` builds, the actual implementation warns `dead_code`, as it isn't used with the tests. This suppresses these errors for `test` builds, code that is not used in normal builds will still warn.